profiles/base/package.use.force

```diff
@@ -91,12 +91,13 @@ sys-libs/libcxx libcxxabi
 # the flag.
 sys-kernel/vanilla-kernel initramfs
 
+# Andreas K. Hüttel <dilfridge@gentoo.org> (2021-07-14)
 # Sergei Trofimovich <slyfox@gentoo.org> (2019-11-06)
```

Comment:
Drop other authors -- pkgdev and other parsers can't handle it, and we have git history anyway. Or just put toolchain@